addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/HomekitThermostatImpl.java

```diff
@@ -46,93 +50,93 @@
  *
  * @author Andy Lintner - Initial contribution
  */
-class HomekitThermostatImpl extends AbstractTemperatureHomekitAccessoryImpl<GroupItem>
-        implements BasicThermostat, GroupedAccessory {
+class HomekitThermostatImpl extends AbstractTemperatureHomekitAccessoryImpl<GroupItem> implements BasicThermostat {
```

Comment:
Can you please annotate all (new) `Impl` classes with `@NonNullByDefault`?
That helps a lot to prevent NPEs to happen and fellow developers immediately know about a potential null access.